pkg/proxy/apis/config/types.go

```diff
@@ -191,23 +191,21 @@ type KubeProxyConfiguration struct {
 }
 
 // ProxyMode represents modes used by the Kubernetes proxy server. Currently, three modes of proxy are available in
-// Linux platform: 'userspace' (older, going to be EOL), 'iptables' (newer, faster), 'ipvs'(newest, better in performance
-// and scalability).
+// Linux platform: 'iptables' (newer, faster), 'ipvs'(newest, better in performance and scalability).
```

Comment:
"iptables" is no longer newer or faster than anything else. I'd just remove these descriptions since I don't like the IPVS one anyway (it doesn't mention the fact that the ipvs proxy is missing a few features still).